static/app/views/settings/featureFlags/newProviderForm.tsx

```diff
@@ -98,13 +108,32 @@ export default function NewProviderForm({
           <SelectField
             required
             label={t('Provider')}
+            onChange={setSelectedProvider}
+            value={selectedProvider}
+            placeholder={t('Select a provider')}
+            defaultValue={'Launchdarkly'}
+            defaultChecked
```

Comment:
What is `defaultChecked` doing?  What does `defaultValue` do if the select does not default to that option.

Reply:
theyre doing nothing. i forgot to delete, but i was experimenting with a bunch of random props to see if anything would work but they didn't